config/application.rb

```diff
@@ -74,6 +74,8 @@ class Application < Rails::Application
       g.test_framework :rspec
     end
 
+    # TODO: this appears to be necessary to suppress deprecations from fog:
+    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
This is actually something that should be added as this is opt-in new behavior that will become the default later. See "3.3 Error handling in transaction callbacks" in the upgrade guide. http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html